builtin/logical/aws/path_config_root.go

```diff
@@ -204,6 +214,41 @@ func (b *backend) pathConfigRootWrite(ctx context.Context, req *logical.Request,
 		return nil, err
 	}
 
+	// Now that the root config is set up, register the rotation job if it required
```

Comment:
Example with AWS Secrets on using common libraries and registering a Rotation Job